extensions/vscode/lsp-client/src/clientWeb.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
```

Comment:
Does MIT require an author's license?

Reply:
Made some changes to the licensing, please check if things are looking correct now

Reply:
Also, you can now use the Wake nightly extension to review the pull request in github.dev!